starter-core/src/test/groovy/io/micronaut/starter/feature/camunda/CamundaSpec.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 original authors
```

Comment:
probably 2021

Reply:
We use currently 2020 all through the project. This will need to be addressed in a separate PR.